### DISCUSSION TITLE
Latest v2 firmware with RTSP

### DISCUSSION THREAD

**Discussion by TheCableGuy99** [gtxaspec/wz_mini_hacks#518] [Q&A]
Hi,

I have an older v2 lying around and want to install the hack. The instructions on the main page state not to upgrade past a specific version if I want to use RTSP, but this version isn't applicable for the v2 camera.

Can anyone tell me the latest version for the v2 camera I can use and still have RTSP?

Also I'm very confused about the install process for this version. Can anyone advise the correct process please?

Thanks.

**Comment by dudududodododedede** [comment]
I have wz_mini RTSP working on v2 firmware version 4.9.8.1002 (since this is the one specifically referenced in the readme as working... other firmware versions may probably work too though)

You need to follow the steps in both of these sections from the readme:
- Setup v2/PanV1
- Installation v2/PanV1
